doc/source/whatsnew/v0.22.0.txt

```diff
@@ -262,7 +262,7 @@ Performance Improvements
 Documentation Changes
 ~~~~~~~~~~~~~~~~~~~~~
 
--
+- Added a reference to `DataFrame.assign` in the concatenate section of the merging documentation
```

Comment:
use ``:func:`DataFrame.assign` ``; add this issue number at the end

Reply:
you can move this to 0.21.1

Reply:
done